spec/p2p/peer.md

```diff
@@ -28,22 +28,33 @@ Both handshakes have configurable timeouts (they should complete quickly).
 
 Tendermint implements the Station-to-Station protocol
 using X25519 keys for Diffie-Helman key-exchange and chacha20poly1305 for encryption.
+
+Previous versions of this protocol suffered from malleability attacks wherea an active man 
```

Comment:
I'm not sure if writing it in this way will bring value in the future? IMO we should write it in the present tense and not link to the past, this mention of previous versions seems like good wording for an adr instead of a spec.